src/neuroconv/datainterfaces/behavior/audio/audiointerface.py

```diff
@@ -0,0 +1,203 @@
+from neuroconv.basedatainterface import BaseDataInterface
+from neuroconv.datainterfaces.behavior.video.videodatainterface import _check_duplicates
+from neuroconv.tools.hdmf import SliceableDataChunkIterator
+from neuroconv.tools.nwb_helpers import (
+    make_or_load_nwbfile,
+)
+from neuroconv.utils import (
+    get_schema_from_hdmf_class,
+    get_base_schema,
+    OptionalFilePathType,
+)
+from pynwb import NWBFile, TimeSeries
+
+from ndx_sound import AcousticWaveformSeries
+
+
+class AudioInterface(BaseDataInterface):
+    """Data interface for writing acoustic recordings to an NWB file."""
+
+    def __init__(self, file_paths: list, verbose: bool = False):
+        """
+        Create the interface for writing acoustic recordings as AcousticWaveformSeries.
+
+        Parameters
+        ----------
+        file_paths: list of FilePathTypes
+            The file paths to the audio recordings in sorted, consecutive order.
+        """
+        suffixes = [suffix for file_path in file_paths for suffix in Path(file_path).suffixes]
+        format_is_not_supported = [suffix for suffix in suffixes if suffix not in [".wav"]]
+        if format_is_not_supported:
+            raise ValueError(
+                "The currently supported file format for audio is WAV file. "
+                f"Some of the provided files does not match this format: {format_is_not_supported}."
+            )
+        self.verbose = verbose
+        super().__init__(file_paths=natsorted(file_paths))
+
+    def get_metadata_schema(self):
+
+        metadata_schema = super().get_metadata_schema()
+        time_series_metadata_schema = get_schema_from_hdmf_class(TimeSeries)
+        metadata_schema["properties"]["Behavior"] = get_base_schema(tag="Behavior")
+        time_series_metadata_schema.update(required=["name"])
+        metadata_schema["properties"]["Behavior"].update(
+            required=["Audio"],
+            properties=dict(
+                Audio=dict(
+                    type="array",
+                    minItems=1,
+                    items=time_series_metadata_schema,
+                )
+            ),
+        )
+        return metadata_schema
+
+    def get_metadata(self):
+
+        behavior_metadata = dict(
+            Audio=[
+                dict(name=f"Audio: {Path(file_path).stem}", description="Acoustic waveform series.")
+                for file_path in self.source_data["file_paths"]
+            ]
+        )
+        metadata = super().get_metadata()
+        metadata["Behavior"] = behavior_metadata
+
+        return metadata
+
+    def add_acoustic_waveform_series(
+        self,
+        nwbfile: NWBFile,
+        metadata: dict,
+        file_path: str,
+        starting_time: float,
+        write_as: Optional[str] = "stimulus",  # "stimulus" or "acquisition"
+        stub_test: bool = False,
+        stub_frames: int = 1000,
+        iterator_options: Optional[dict] = None,
+        compression_options: Optional[dict] = None,
+    ):
```

Comment:
I would prefer to follow the past modularization structure where helper functions like this exist in the `neuroconv.tools` as a sub-module independent of the interfaces so they can be harnessed more generally